source/runtime_gui.cpp

```diff
@@ -3204,12 +3204,28 @@ void reshade::runtime::draw_variable_editor()
 					const int ui_max_val = variable.annotation_as_int("ui_max", 0, ui_type == "slider" ? 1 : std::numeric_limits<int>::max());
 					const int ui_stp_val = std::max(1, variable.annotation_as_int("ui_step"));
 
+#define DESCRIPTION_DEFAULT_STRING "%d"
+					const size_t length_of_description = sizeof(DESCRIPTION_DEFAULT_STRING) - 1;
+					const size_t max_description_length = 11; // space + 10 characters
+					const size_t max_chars_for_description = max_description_length + 1; // +1 for \0
+					char description[max_chars_for_description] = DESCRIPTION_DEFAULT_STRING;
+#undef DESCRIPTION_DEFAULT_STRING
+
+					const std::string_view description_view = ui_type == "slider" ? variable.annotation_as_string("ui_slider_desc")
+															: ui_type == "drag"   ? variable.annotation_as_string("ui_drag_desc")
+															: "";
```

Comment:
I'd consider naming the annotation `ui_units` uniformly, since that's what it should most likely be used for.

Reply:
yeah makes sense